docs/diagrams/HelpClassDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+ skinparam backgroundColor #ffffff
+ skinparam classAttributeIconSize 0
+ hide circle
+
```

Comment:
Probably can include the HelpWindow class in the diagram since it has some dependency on the HelpCommand

Reply:
I have made the necessary changes! Thank you 💯